### ISSUE TITLE
Additional fields for sending metric data via HTTP/JSON

### ISSUE THREAD

**Issue by genachka** [ngrossmann/momo#1]
Cool project! I'm a big fan of Couchbase and starting out with Grafana. I'm receiving data from IoT devices that are coming in to Kafka and then Spark via Streams. From Spark I'm planning on sending to Momo via the HTTP as JSON post. But in Grafana I need to have flexibility to build query for graphs for additional details. From what I see you're currently supporting only:

```
{
  name: "time.series.name",
  timestamp: <timestamp in milliseconds since the epoch>
  value: <long value>
}
```

I'd like to have **sensor type** and **location** as well:

```
{
  name: "time.series.name",
  timestamp: <timestamp in milliseconds since the epoch>
  value: <long value>,
  type: "sensor type string",
  location: "device location string"
}
```

Would it be possible to add those?

**Comment by ngrossmann** [comment]
I assume those fields don't change over time, i.e. they are an attribute of the time-series not each data point, right?

**Comment by genachka** [comment]
Correct. Metadata that would be helpful in filtering and designing specific dashboards.
